modules/src/core/ics04_channel/handler/acknowledgement.rs

```diff
@@ -195,23 +193,12 @@ mod tests {
                 msg: msg.clone(),
                 want_pass: false,
             },
-            Test {
-                name: "Processing fails because the port does not have a capability associated"
-                    .to_string(),
-                ctx: context.clone().with_channel(
-                    PortId::default(),
-                    ChannelId::default(),
-                    source_channel_end.clone(),
-                ),
-                msg: msg.clone(),
-                want_pass: false,
-            },
             Test {
                 name: "Good parameters".to_string(),
                 ctx: context
                     .with_client(&ClientId::default(), client_height)
                     .with_connection(ConnectionId::default(), connection_end)
-                    .with_port_capability(packet.destination_port.clone())
+                    .with_port(packet.destination_port.clone())
```

Comment:
IIUC, I don't think we need to set a port mapping to test handshakes, so we should be able to remove most of them.